ci-operator/step-registry/openshift/microshift/infra/rpm-install-aws/openshift-microshift-infra-rpm-install-aws-commands.sh

```diff
@@ -33,6 +33,16 @@ cd ~/microshift
 EOF
 chmod +x /tmp/install.sh
 
+if "${SRC_FROM_GIT}"; then
+  mkdir -p /go/src/github.com/openshift
+  branch=$(echo ${JOB_SPEC} | jq -r '.refs.base_ref')
+  # MicroShift repo is recent enough to use main instead of master.
+  if [ "${branch}" == "master" ]; then
+    branch="main"
+  fi
+  git clone https://github.com/openshift/microshift -b $branch /go/src/github.com/openshift/microshift
+fi
```

Comment:
This clone is more logical, however would also be good to comment it and describe exactly what's going on.

Reply:
Is this enough? https://github.com/openshift/release/pull/56264/files#diff-7980835a19d56cc43b6c93d39084cf9a5d10171611e61c521d4a0a77708e0134R17-R19